src/components/calendar/Day.vue

```diff
@@ -10,16 +11,63 @@
 	}
 
 	const props = defineProps<Props>()
+	const emit = defineEmits(['intersects'])
 
-	// TODO: handle relative index reactive click from Increment component
-	// and bubble up to Calendar component as a unified (i.e., selected) range
+	let dragging: boolean = false
+
+	// Using for O(1) even tho vals are not semantically important
+	// (i.e., .exists() would work just as fine)
+	let intersections = reactive(new Map())
+
+	function mousedown(event: Event) {
+		dragging = true
+		// Skip default behavior to prevent edit cursor in Safari
+		// src: https://stackoverflow.com/a/9743380/1431900
+		event.preventDefault()
+	}
+
+	function mouseup(event: Event) {
+		dragging = false
+	}
+
+	function mouseleave(event: Event) {
+		dragging = false
+	}
+
+
+	// MARK: Intersection & child events
+	// These are mouse events we explicitly listen to the child for
+	// to avoid intersection calculations
+
+	function childMouseenter(index: number) {
+		if (!dragging) { return }
+		// Toggle
+		const selected = intersections.get(index)
+		intersections.set(index, !selected)
+	}
+
+	function childMousedown(index: number) {
+		intersections.set(index, !intersections.get(index))
```

Comment:
Might want to check to see if index exists first, otherwise you might be trying to invert undefined.